terraform.tfvars.example

```diff
@@ -0,0 +1,3 @@
+aws_access_key="XXXXXXXXXXXX"
+aws_secret_key="XXXXXXXXXXXX"
+aws_key_pair="digital-eq-keypair"
```

Comment:
Is this key pair used by this deployment? The referenced keypair above is `pre-prod.pem` ?